modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java

```diff
@@ -761,6 +780,18 @@ private Page<Document> _getDocumentsPage(
 					GetterUtil.getLong(document.get(Field.ENTRY_CLASS_PK)))));
 	}
 
+	private Page<Document> _getGroupDocumentsRatedByMePage(
+			long groupId, Pagination pagination)
+		throws Exception {
+
+		return Page.of(
+			_toDocuments(
+				_dlFileEntryService.getFileEntries(
+					groupId, 1.0, pagination.getStartPosition(),
+					pagination.getEndPosition())),
+			pagination, _dlFileEntryService.getFileEntriesCount(groupId, 1.0));
+	}
+
```

Comment:
The rating value goes from 0.0 to 1.0, this means we should use the minimum value here, which is probably 0.1. (For ratings from 1 to 5 the minimum is 0.2) However, to go really safe I would change the GTE condition to GT and use the value 0.0.

Reply:
The values used in the tests are arbitrary. We don't need to check that ratings works (that's done by other tests), only that we get the expected results with a particular choice of values.

About GTE vs GT, I've chosen GTE to avoid forcing API consumers to use weird values like "0.19" when looking for specific ratings (e.g.: 0.19 would need to be used when looking for a "2 or more stars").

Reply:
Please notice the requested change is not for a test.

Reply:
Sorry, my bad. For whatever reason I thought you where talking about the tests. I agree on using a lower value. Not about the GT vs GTE thing though.

Reply:
We both agreed on the GT vs GTE issue in a Slack conversation. Leaving it as it is.